### DISCUSSION TITLE
Is it possible to attach or link files?

### DISCUSSION THREAD

**Discussion by Kwonsoonchol** [flowsforapex/apex-flowsforapex#643] [Q&A]
I wonder if the file attachment function or the link function to the process activity will be supported in the future. As our company's official bpmn tool, support for file attachment and link functions is an important factor in deciding whether to use this application, so I'm curious about your future plans.

For example, in the case of printouts related to the task, a separate link or file attachment is made to be useful for reference.   I understand that the function of flows4apex is only about the function of calling an apex page.

**Comment by rallen2010** [comment]
Not sure if it's too late to pick this up.... Is the requirement to attach a file or a link to a BPMN activity at runtime, or to an activity in the process modeler?
